misc/slack-notification/on-approve.sh

```diff
@@ -6,3 +6,5 @@ if [[ -n "${ENV0_REVIEWER_NAME}" ]]; then
     __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
     source "${__dir}"/slack.sh "$MESSAGE"
 fi
+
+touch apply
```

Comment:
missing newline at end of file - please adjust that setting in your IDE 🙏 

Also - can we go with a more descriptive name like `applied` or `apply_ran_successfully` ?